numprocs=8

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v6_batch1/v6_batch1_4_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v6_batch1/v6_batch1_4_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v6_batch1/v6_batch1_4_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v6_batch1/v6_batch1_4_cfg.json ... 

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v6_batch1/v6_batch1_4_cfg.json ... 
Loading simConfig...
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v6_batch1/v6_batch1_4_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v6_batch1/v6_batch1_4_cfg.json ... 
Loading simConfig...

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v6_batch1/v6_batch1_4_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v6_batch1/v6_batch1_netParams.py
Importing netParams from ../data/v6_batch1/v6_batch1_netParams.py
Importing netParams from ../data/v6_batch1/v6_batch1_netParams.py
Importing netParams from ../data/v6_batch1/v6_batch1_netParams.py
Importing netParams from ../data/v6_batch1/v6_batch1_netParams.py
Importing netParams from ../data/v6_batch1/v6_batch1_netParams.py

Start time:  2021-08-19 14:34:36.468963
Importing netParams from ../data/v6_batch1/v6_batch1_netParams.py
Importing netParams from ../data/v6_batch1/v6_batch1_netParams.py

Creating network of 55 cell populations on 8 hosts...
  Number of cells on node 5: 3918 
  Number of cells on node 1: 3919 
  Number of cells on node 6: 3918 
  Number of cells on node 0: 3919 
  Done; cell creation time = 23.65 s.
Making connections...
  Number of cells on node 7: 3918 
  Number of cells on node 3: 3918 
  Number of cells on node 2: 3918 
  Number of cells on node 4: 3918 
  Number of connections on node 2: 109144 
  Number of synaptic contacts on node 2: 762118 
  Number of connections on node 0: 107750 
  Number of synaptic contacts on node 0: 754076 
  Number of connections on node 4: 109733 
  Number of synaptic contacts on node 4: 766854 
  Number of connections on node 3: 109570 
  Number of synaptic contacts on node 3: 764790 
  Number of connections on node 1: 108264 
  Number of synaptic contacts on node 1: 758484 
  Number of connections on node 6: 108706 
  Number of synaptic contacts on node 6: 761730 
  Number of connections on node 7: 109520 
  Number of synaptic contacts on node 7: 763992 
  Number of connections on node 5: 108527 
  Number of synaptic contacts on node 5: 759804 
  Done; cell connection time = 1260.64 s.

Gathering data...
  Done; gather time = 91.01 s.

Analyzing...
Plotting connectivity matrix...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'",), <traceback object at 0x7f86a461e348>)
Plotting connectivity matrix...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'",), <traceback object at 0x7fec85542348>)
Plotting connectivity matrix...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'",), <traceback object at 0x7f34cd8d52c8>)
Plotting connectivity matrix...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'",), <traceback object at 0x7fc486731308>)
Plotting connectivity matrix...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'",), <traceback object at 0x7f19113812c8>)
Plotting connectivity matrix...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'",), <traceback object at 0x7f1d9b7b1348>)
Plotting connectivity matrix...
  There was an exception in plotConn(): 
    'Network' object has no attribute 'allCells' 
    (<class 'AttributeError'>, AttributeError("'Network' object has no attribute 'allCells'",), <traceback object at 0x7efba8676308>)
  Cells: 31346
  Connections: 871214 (27.79 per cell)
  Synaptic contacts: 6091848 (194.34 per cell)
Plotting connectivity matrix...
Saving figure data as ../data/v6_batch1/v6_batch1_4_pop_numConns_matrix.json ... 
